yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -914,3 +974,129 @@ def run(self, info):
         if not has_thumbnail:
             self.to_screen('There aren\'t any thumbnails to convert')
         return files_to_delete, info
+
+
+class FFmpegRemoveChaptersPP(FFmpegPostProcessor):
+    SUPPORTED_SUBS = ('srt', 'ass', 'vtt')
+
+    def __init__(self, downloader, regexes, force=False, force_keyframes=False):
+        FFmpegPostProcessor.__init__(self, downloader)
+        self._regexes = [re.compile(regex) if isinstance(regex, str) else regex for regex in regexes]
+        self._force_cut = force
+        self._force_keyframes = force_keyframes
+
+    def run(self, info):
+        ranges_to_cut = self._remove_chapters_from_infodict(info.get('chapters') or [], regexes=self._regexes)
+        if not ranges_to_cut:
+            if not info.get('chapters'):
+                self.to_screen('Chapter information is unavailable')
+            else:
+                self.to_screen('There are no chapters matching the regex')
+            return [], info
+
+        if not info.get('__real_download'):
+            if self._force_cut:
+                self.report_warning('Removing chapters multiple times may cut out unintended parts of the video')
+            else:
+                self.report_warning(
+                    f'Skipping {self.pp_key()} since the video was already downloaded. '
+                    'Use --force-remove-chapters to remove the chapters anyway')
+                return [], info
```

Comment:
TODO: Even if not `__real_download`, if the duration in the infodict and that obtained from ffprobe matched, removing chapters should be safe

Reply:
Yes, that is the point. If the durations obtained from ffprobe and from the website match, we can safely run the PP without a --force option. In fact, implementing this should completely get rid of the need for --force switches like you suggest below

Reply:
In case there was a miscommunication, I put the TODO comment to remind myself to implement the duration check - it wasn't aimed at you